shell.nix

```
@@ -51,12 +51,16 @@ pkgs.mkShell {
 
       # For Cabal to clone git repos
       pkgs.buildPackages.git
+
+      # Clone verilog-ethernet
+      pkgs.verilog-ethernet
```

```suggestion
      # HDL dependencies
      pkgs.verilog-ethernet
```

let's just make it a list to store our raw HDL deps